.circleci/config.yml.jinja

```diff
@@ -218,6 +241,26 @@ jobs:
                   path: test_logs
     {%- endfor %}
 
+    {%- for profile in medium_curves_tests.keys() | sort %}
+    test--{{profile}}:
+        resource_class: large
+        docker:
+            - image: codaprotocol/coda:toolchain-bc9f6e8cb1f2132cb964b4a3caa3aecee67b1826
+        steps:
+            - checkout
+            - run: git submodule sync && git submodule update --init --recursive
+            {%- for test in medium_curves_tests[profile] %}
+            - run:
+                  name: Running test -- {{profile}}:{{test}}
+                  command: ./scripts/skip_if_only_frontend.sh bash -c 'source ~/.profile && ./scripts/test.py run "{{profile}}:{{test}}"'
+                  {%- if profile in medium_curve_profiles %}
```

Comment:
This check seems unnecessary here since you are iterating through the `medium_curves_tests`.

Reply:
`medium_curve_profiles` includes only profiles with `proof_level=full`. I should probably rename that